src/Files.App/Helpers/UIFilesystemHelpers.cs

```diff
@@ -48,7 +50,7 @@ public static async void CutItem(IShellPage associatedInstance)
 
                 try
                 {
-                    var dispatcherQueue = DispatcherQueue.GetForCurrentThread();
+                    var threadingService = Ioc.Default.GetRequiredService<IThreadingService>();
```

Comment:
Can you add a comment explaining why this is needed? It's helpful to prevent someone from changing this in the future 🙂

Reply:
Could you confirm that Ctrl+X is currently _not_ working for you on main?

Reply:
It's working for me on `main` but Windows 10 and different specs could be at play.

Reply:
Should I merge?

Reply:
Mmm better to wait to merge this one then b/c this change should not be necessary (and I could swear it used to work).